CorsixTH/Lua/announcer.lua

```diff
@@ -0,0 +1,157 @@
+-- NB: Every tick is 30ms, so 2000 ticks is 1 minute
+
+local AnnouncementPriority = {
+  Critical = 0,
+  High = 1,
+  Normal = 2,
+  Low = 3
+}
+
+strict_declare_global "AnnouncementPriority"
+_G["AnnouncementPriority"] = AnnouncementPriority
+
+local default_announcement_priority = AnnouncementPriority.Normal
+
+local default_announcement_decay_time = {
```

Comment:
We can tweak values here. I don't think they make sense right now, but its a start.